packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -358,7 +358,8 @@ function Routes() {
         options={{
           searchStringToObject: parse,
           objectToSearchString: stringify,
-          updateType: 'replaceIn' // replace just a single parameter when updating query-state, leaving the rest as is
+          updateType: 'replaceIn', // replace just a single parameter when updating query-state, leaving the rest as is
+          removeDefaultsFromUrl: true
```

Comment:
What's the reason on adding this, is it so we don't have an empty `?amount=` in the URL?

Reply:
yes, it removes the defaults (for amount it's the empty string)